### PR TITLE
[CRYPTEXT] Make extension localizable

### PR DESCRIPTION
**Pull request by peterooch**
Localize error message and shell file extension description.
Added Hebrew resources.
Added Icon.

Follow up of #1343 

Icon is the tango icon named "certificate", if anybody has a idea for more fitting icon please recommend it.
JIRA issue: [CORE-15736](https://jira.reactos.org/browse/CORE-15736)

cryptext_loc

### CODE REVIEW

File: dll/shellext/cryptext/lang/en-US.rc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     English (United States) resource translation
+ * COPYRIGHT:   Copyright 2019 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+STRINGTABLE
+BEGIN
+    IDS_INVALIDFILE "This is not a valid certificate file"
```
Comment (HBelusca) [inline dll/shellext/cryptext/lang/en-US.rc]: Sentence ending with a dot?

File: dll/shellext/cryptext/lang/he-IL.rc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Hebrew resource translation
+ * COPYRIGHT:   Copyright 2019 Baruch Rutman (peterooch at gmail dot com)
+ */
+
+LANGUAGE LANG_HEBREW, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_INVALIDFILE "קובץ זה הוא לא קובץ אישורי הצפנה תקין"
```
Comment (HBelusca) [inline dll/shellext/cryptext/lang/he-IL.rc]: (I suppose this is the same case in Hebrew?)

File: dll/shellext/cryptext/lang/en-US.rc (hunk 1 above)
Comment (learn-more) [reply dll/shellext/cryptext/lang/en-US.rc]: Why?

Comment (HBelusca) [reply dll/shellext/cryptext/lang/en-US.rc]: That's the string that is displayed in the message box.
You sound similar to as if I was asking to put a capital letter at the beginning of the sentence, and you ask "why?".

Comment (gonzoMD) [reply dll/shellext/cryptext/lang/en-US.rc]: Or better an exclamation mark

**Comment by JaredSmudde** [comment]
Icon looks fine.

**Pull request merged**